### ISSUE TITLE
Screen reader user cannot navigate to the list of country options using the swiping motion

### ISSUE THREAD

**Issue by JordHemsworth** [alphagov/accessible-autocomplete#773]
Following [priority 1 (P1) accessibility audit issue for CRS/FATCA registration (9108)](https://github.com/hmrc/accessibility-audits-external/issues/9108) it was discovered a screen reader can not navigate the country list without first clicking on to it.

Tested with VoiceOver on iOS (version 18)

Currently using
- play-frontend-hmrc - 11.12.0
- No polyfill 
- Aria-controls over aria-owns
